Makefile

```diff
@@ -0,0 +1,7 @@
+lint:
```

Comment:
missing license header

targets that aren't produced by their recipes ought to be labelled as such with [the special `.PHONY` target](https://www.gnu.org/software/make/manual/html_node/Phony-Targets.html)